GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListViewModel.swift

```diff
@@ -11,50 +11,51 @@ import Foundation
 import Combine
 
 class DogBreedsListViewModel: ObservableObject {
-        @Published var dogBreeds = [DogBreed]()
-        private let dogBreedsService: DogBreedsServing
-        
-        @Published private(set) var hasError: Bool = false
-        @Published private(set) var isLoading: Bool = false
-        @Published private(set) var isImageLoading: Bool = false
-        @Published private(set) var currentPost: [DogBreed]?
-        
-        private(set) var imageLoader = ImageLoader()
-        
-        private var cancellables = Set<AnyCancellable>()
-        
-        init(dogBreedsService: DogBreedsServing) {
-            self.dogBreedsService = dogBreedsService
-        }
-        
-        deinit {
-            cancellables.removeAll()
-        }
-        
-        func initialFetch() {
-            getDogBreeds()
-        }
-        
-        func finishImageLoading() {
-            isImageLoading = false
-        }
-        
-        func getDogBreeds() {
-            dogBreedsService
-                .getDogBreeds()
-                .receive(on: DispatchQueue.main)
-                .sink(
-                    receiveCompletion: { [weak self] resp in
-                        if case .failure = resp {
-                            self?.hasError = true
-                            self?.isLoading = false
-                            self?.isImageLoading = false
-                        }
-                    }, receiveValue: { [weak self] val in
-                        self?.currentPost = val
+    private let dogBreedsService: DogBreedsServing
+    
+    @Published var dogBreeds = [DogBreed]()
+    
+    @Published private(set) var hasError: Bool = false
```

Comment:
This pr does not concentrate on:

1. Error
2. Loading
3. Image loading (it was needed in reddit vm)
4. Current post (it was needed in reddit vm)

Please remove these properties.

Reply:
okey, I. will fix.